.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,29 @@
 <!--  Thanks for sending a pull request!  Here are some tips for you:
-1. Consider creating this PR as draft: https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#consider-opening-your-pull-request-as-draft
+1. Consider creating this PR as a draft: https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#consider-opening-your-pull-request-as-draft
 2. Follow the instructions for writing a release note from k8s: https://git.k8s.io/community/contributors/guide/release-notes.md
 -->
 
 ### What this PR does
-Before this PR:
+#### Before this PR:
 
-After this PR:
+#### After this PR:
 
-<!-- (optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*: -->
-Fixes #
+### References
+<!-- optional,
+  Use `Fixes #<issue number>(, Fixes #<issue_number>, ...)` format, to close the issue(s) when PR gets merged.
+  Use `Partially addresses #<issue number>` to link an issue without closing it when the PR merges.
+  
+- Fixes #
+- Partially addresses #
+-->
+<!-- optional,
+  Link the VEP tracking issue (https://github.com/kubevirt/enhancements/issue/<vep_tracking_issue_number>) and the 
+  VEP PR (https://github.com/kubevirt/enhancements/pull/<vep_pr_number>) if this PR is implementing one.
+  For additional info about VEP PR and VEP tracking issue, see https://github.com/kubevirt/enhancements#process
```

Comment:
I find the term `VEP PR` confusing, since the VEP design is just a merged file which could be updated with many PRs.

WDYT about
```suggestion
  VEP design (e.g. https://github.com/kubevirt/enhancements/blob/main/veps/sig-compute/10-dra-devices/vep.md, or a VEP PR if the design is not merged yet) if this PR is implementing one.
  For additional info about VEP PR and VEP tracking issue, see https://github.com/kubevirt/enhancements#process
```